### PR TITLE
Extract and store requirement field in commandLineTool

### PR DESCRIPTION
**Pull request by Cenxn**
- Created  a Requirement model in the database
- Read the requirement field, extract the class field, and store it as a JSON payload. (linked with task table)

Since CWLSchemaValidator automatically verifies that the file conforms to the cwl syntax and the required data types, storing it as a JSON payload shouldn't cause serious problems in type constraint.

### CODE REVIEW

**Comment by anisahn09** [comment]
I just realised workflow can also has "requirements" filed. Should we separate the model or how do you think? @Cenxn

**Comment by waleedev01** [comment]
yes, @anisahn09 we will probably have to add requirements and hints json fields to job as well.

**Comment by anisahn09** [comment]
> yes, @anisahn09 we will probably have to add requirements and hints json fields to job as well.

I see, is there also "hints" for task?

**Comment by anisahn09** [comment]
> yes, @anisahn09 we will probably have to add requirements and hints json fields to job as well.

Also if there will be two requirement tables, maybe we should name them in the same way. Like Requirement_Task and Requirement_Job, or something like that. How do you think? @Cenxn @waleedev01

**Comment by anisahn09** [comment]
> > > yes, @anisahn09 we will probably have to add requirements and hints json fields to job as well.
> > 
> > 
> > Also if there will be two requirement tables, maybe we should name them in the same way. Like Requirement_Task and Requirement_Job, or something like that. How do you think? @Cenxn @waleedev01
> 
> I guess we are using the requirements json field, so we don't need to manage this table name issue.

Wait so there will be only one `Requirement` table? or two table, one for task, one for job?

**Comment by waleedev01** [comment]
> > > > yes, @anisahn09 we will probably have to add requirements and hints json fields to job as well.
> > > 
> > > 
> > > Also if there will be two requirement tables, maybe we should name them in the same way. Like Requirement_Task and Requirement_Job, or something like that. How do you think? @Cenxn @waleedev01
> > 
> > 
> > I guess we are using the requirements json field, so we don't need to manage this table name issue.
> 
> Wait so there will be only one `Requirement` table? or two table, one for task, one for job?

Requirements will be a json payload field in both tasks and Job, not a table.

File: analytics_automated/models.py [hunk 1]
```diff
@@ -244,6 +244,19 @@ class Meta:
         app_label = 'analytics_automated'
 
 
+class Requirement(models.Model):
```
Comment (anisahn09) [inline analytics_automated/models.py:247]: isn't this creating a new table for task only? @waleedev01

Comment (Cenxn) [reply analytics_automated/models.py:247]: Yes in this case I create a table called Requirement to aviod long reuqirement json payload. We should now use attribute instead. I will cancel this request and resubmit one

**Pull request closed without merging**